app/registries/foreman/access_permissions.rb

```diff
@@ -807,4 +807,13 @@
                    :ssh_keys => [:destroy],
                    :"api/v2/ssh_keys" => [:destroy]
   end
+
+  permission_set.security_block :personal_access_tokens do |map|
+    map.permission :view_personal_access_tokens,
```

Comment:
this does not seem to be used anywhere, I'd expect this in view where tokens are listed, so we make sure that only user who owns tokens or another user with this permission can see them